RepoMan.cmake

```
@@ -9,9 +9,21 @@ This module handles project dependencies.
 Usage
 ^^^^^
 
-Including the module is suficient. It will automatically look for a dependencies file in ``PROJECT_SOURCE_DIR`` and resolve the dependencies recursively.
+Including the module is sufficient. It will automatically look for a dependencies file in ``PROJECT_SOURCE_DIR`` and resolve the dependencies recursively.
 
 .. code-block:: cmake
+  # Optional: use workspace instead of default FetchContent directories
+  set(REPOMAN_DEPENDENCIES_USE_WORKSPACE ON CACHE BOOL "")
+
+  # Optional: set a custom name and/or location for the workspace
+  set(REPOMAN_DEPENDENCIES_WORKSPACE "../" CACHE PATH "")                     # use automatically generated directory name next to current project
```

I am unhappy with this asymmetry. A user may want all dependencies side by side directly. I propose to change the behavior such that the default value of `REPOMAN_DEPENDENCIES_WORKSPACE` is  `${PROJECT_SOURCE_DIR}/../$( basename ${PROJECT_SOURCE_DIR}-dependencies)`.

```suggestion
  set(REPOMAN_DEPENDENCIES_WORKSPACE "../" CACHE PATH "")                     # put dependencies next to current project
```

---

Below it is described differently, hence maybe changing the comment is all we need.